<?xml version="1.0"?>
<launch>
    <include file="$(find ur10_moveit_config)/launch/ur10_moveit_planning_execution.launch">
        <arg name = "sim" value="true"/>
    </include>

    <node name = "project_ariac_example" pkg="project_ariac" type = "project_ariac_node" output = "screen"/>
<!--         <param name = "elbow_joint" value = "0" />
        <param name = "linear_arm_actuator_joint" value = "0" />
        <param name = "shoulder_lift_joint" value = "0" />
        <param name = "shoulder_pan_joint" value = "0" />
        <param name = "wrist_1_joint" value = "0" />
        <param name = "wrist_2_joint" value = "0" />
        <param name = "wrist_3_ijoint" value = "0" />
        <param name = "z_offSet_pickUp" value = "0.026" />
        <param name = "planner" value ="RRTConnectkConfigDefault"/>
        <param name = "planning_time" value="2.5"/>
        <parma name = "planning_attempt" value="20"/> -->
</launch>
